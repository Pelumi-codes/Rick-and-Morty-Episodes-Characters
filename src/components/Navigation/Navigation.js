import React from "react";
import { Box, Text } from "@chakra-ui/react";

const Navigation = () => {
  return (
    <Box>
      <Text
        fontWeight="800"
        fontSize="60px"
        textAlign="center"
        mt="2%"
        letterSpacing="wider"
        textShadow="1.5px 1.5px 2px grey"
      >
        The Rick and Morty
      </Text>
    </Box>
  );
};

export default Navigation;
