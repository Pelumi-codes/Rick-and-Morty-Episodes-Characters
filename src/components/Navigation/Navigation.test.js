import React from "react";
import renderer from "react-test-renderer";

import Navigation from "./Navigation";
it("renders list of characters", () => {
  const tree = renderer.create(<Navigation />).toJSON();
  expect(tree).toMatchSnapshot();
});
