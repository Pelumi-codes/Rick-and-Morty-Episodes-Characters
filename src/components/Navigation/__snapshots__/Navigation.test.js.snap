// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders list of characters 1`] = `
<div
  className="css-0"
>
  <p
    className="chakra-text css-1ffmg5n"
  >
    The Rick and Morty
  </p>
</div>
`;
